File: Proyectos Android/RetrofitTuto/app/src/main/java/com/example/retrofittuto/entities/ApiClient.java
package com.example.retrofittuto.entities;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiClient {
    @GET("pokemon/")
    Call<ApiResponse> getPokemons(@Query("limit") int limit, @Query("offset") int offset);
}

File: Proyectos Android/RetrofitTutoKotlin/app/src/main/java/com/example/retrofittutokotlin/entities/ApiClient.kt
package com.example.retrofittutokotlin.entities

import retrofit2.http.GET
import retrofit2.http.Query
import retrofit2.Call

interface ApiClient {
    @GET("pokemon/")
    fun getPokemons(@Query("limit") limit : Int, @Query("offset") offset: Int) : Call<ApiResponse>
}